### ISSUE TITLE
rust installation failure on 0.3.1

### ISSUE THREAD

**Issue by mooreniemi** [bminixhofer/nnsplit#9]
Followed your Rust instructions and am hitting this failure:

```
   Compiling nnsplit v0.3.1
error: couldn't read /home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/nnsplit-0.3.1/src/../../models.csv: No such file or directory (os error 2)
  --> /home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/nnsplit-0.3.1/src/model_loader.rs:10:23
   |
10 |         let raw_csv = include_str!("../../models.csv");
   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

error: could not compile `nnsplit`.
```

When I look inside `.cargo/registry/src/github.com-1ecc6299db9ec823/nnsplit-0.3.1/` I see there is no `models.csv`. If I manually add it by copying down [raw](https://raw.githubusercontent.com/bminixhofer/nnsplit/master/models.csv), it still fails on the path.

**Comment by bminixhofer** [comment]
Hi, thanks for reporting! Was kind of a silly mistake, I just pushed a fix. I'll do a release when it passes CI.

**Comment by bminixhofer** [comment]
This is now fixed as of version 0.3.4
